analysis/cross_partition_combiners.py

```diff
@@ -282,33 +278,59 @@ class CrossPartitionCombiner(pipeline_dp.combiners.Combiner):
     # 1. The sum of non dp metrics, which is used for averaging of error
     # metrics.
     # 2. metrics.UtilityReport contains error metrics.
-    AccumulatorType = Tuple[Tuple, metrics.UtilityReport]
-
-    def __init__(self, dp_metrics: List[pipeline_dp.Metrics],
-                 public_partitions: bool):
+    AccumulatorType = Tuple[Tuple, metrics.UtilityReport, Tuple[float]]
+
+    @staticmethod
```

Comment:
Done
Should we maybe then move them into dedicated file to make it visible where to find them? Because these functions are part of the public API and we need to make easy to find them and use them.